### ISSUE TITLE
Center alignment breaks with SVG image

### ISSUE THREAD

**Issue by mjasperse** [py-pdf/fpdf2#1000]
Thanks for this extremely useful project! Minor issue I encountered when swapping from using a raster image to an SVG image is that the  `x=Align.C` feature does not appear to be implemented for vector images.

Describe the bug

**Error details**

```
  File "[...].py", line 42, in add_logo
    self.image('logo.svg', h=h, x=Align.C)
  File "C:\Users\Martijn Jasperse\AppData\Roaming\Python\Python310\site-packages\fpdf\fpdf.py", line 222, in wrapper
    return fn(self, *args, **kwargs)
  File "C:\Users\Martijn Jasperse\AppData\Roaming\Python\Python310\site-packages\fpdf\fpdf.py", line 3884, in image
    return self._vector_image(img, x, y, w, h, link, title, alt_text)
  File "C:\Users\Martijn Jasperse\AppData\Roaming\Python\Python310\site-packages\fpdf\fpdf.py", line 4068, in _vector_image
    path.transform = path.transform @ drawing.Transform.translation(x, y)
  File "C:\Users\Martijn Jasperse\AppData\Roaming\Python\Python310\site-packages\fpdf\drawing.py", line 1013, in __matmul__
    e=self.e * other.a + self.f * other.c + other.e,
TypeError: unsupported operand type(s) for +: 'float' and 'Align'
```

**Minimal code**
Please include some minimal Python code reproducing your issue:
```python
from fpdf import FPDF, Align
pdf = FPDF()
pdf.add_page()
pdf.image('logo.svg', x=Align.C)
```

**Environment**
Please provide the following information:
* **Operating System**: Windows 11
* **Python version**: 3.10.11
* **`fpdf2` version used**: fpdf2==2.7.6

**Comment by gmischler** [comment]
Welcome to fpdf2, @mjasperse!
Thanks for the concise and easily reproducible bug report.

Raster and vector image do have some inherent differences in how they need to be treated.
But we also have a few unnecessary differences, and this is one of those. I've just recently done some minor cleanup in the area (enabling `keep_aspect_ratio` for SVG files), but there's obviously more that needs to be done there. I'll see if I can find the time to have another look soon.

**Comment by gmischler** [comment]
I've just merged #1003, which fixes the problem. The fix will be part of the next release 2.7.7.
If you want to try it before then you can install it with:

`pip install git+https://github.com/py-pdf/fpdf2.git@master`
